daisy_workflows/export/disk_resizing_mon.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+METADATA_URL="http://metadata/computeMetadata/v1/instance"
+
+MAX_SIZE=$1
+BUFFER_SIZE=25
+INTERVAL=10
+
+echo "Max disk size ${MAX_SIZE}GB, buffer size ${BUFFER_SIZE}GB, starting monitoring available disk buffer every ${INTERVAL}s..."
+while [ 1 ]
+do
+  sleep ${INTERVAL}
+
+  # Check whether available buffer space is lower than threshold.
+  AVAILABLE_BUFFER=$(df -BG /dev/sdc --output=avail | sed -n 2p)
+  AVAILABLE_BUFFER=${AVAILABLE_BUFFER%?}
+  if [ ${AVAILABLE_BUFFER} -ge ${BUFFER_SIZE} ]; then
```

Comment:
use double bracket syntax in bash, e.g. `if [[ $AVAILABLE_BUFFER -ge $BUFFER_SIZE ]]; then`

Reply:
I followed the pattern in the codebase which usually use single '['. It may also have better compatibility. May I know why '[[' is prefered?

Reply:
[ and [[ are different functionally, and [[ is much newer and follows bash conventions in an easier to understand way. it's also recommended in our style guide: 
https://google.github.io/styleguide/shell.xml#Test,_%5B_and_%5B%5B